pkg/monitor/intervalcreation/node.go

```diff
@@ -201,7 +201,10 @@ func readinessFailure(logLine string) monitorapi.Intervals {
 			Condition: monitorapi.Condition{
 				Level:   monitorapi.Info,
 				Locator: containerRef.ToLocator(),
```

Comment:
doesn't this give you the pod and container name?

Reply:
yes.  I see I can use allowedRepeatedEventFns that can make use of the namespace, pod, etc.